test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -122,61 +115,37 @@ public Task Localization_CustomCulture()
                 "Hello Hello Hello Hello");
         }
 
-        [ConditionalFact]
```

Comment:
I would say paying attention to the original `RuntimeFlavor` order of these tests is not super valuable, because they were very muddled already. Take this test for example, as it was written it would only run if the OS was non-windows and the framework was CLR, meaning that it will never run.